pkg/metadata/labels/types.go

```diff
@@ -0,0 +1,23 @@
+package labels
+
+const ODHAppPrefix = "app.opendatahub.io"
+
+// K8SCommon keeps common kubernetes labels [1]
+// used across the project.
+// [1] (https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels)
+var K8SCommon = struct {
```

Comment:
try to think why to create struct instead of using a const here.

e.g someday we want to add another new label `app.kubernetes.io/managed-by: Wen`
then update it to
```
var K8SCommon = struct {
    PartOf string
    ManagedBy string
}{
	PartOf: "app.kubernetes.io/part-of",
    ManaagedBy: "app.kubernetes.io/managed-by",
}
```
call with `labels.K8SCommon.ManagedBy:"Wen"`

VS
```
var K8SCommonPartOf = "app.kubernetes.io/part-of"
var K8SCommonManagedBy = "app.kubernetes.io/managed-by"
```

Reply:
At this point it's for better grouping and discovery by IDE. labels.<space>K8SCommon.<space>PartOf.

I just thought it's a neat convention worth following. Makes code easier to work with in my eyes. But I'm open to discussion. I thought it would be better to just show the code instead of trying to discuss the idea on e.g. slack.